conftest.py

```diff
@@ -21,7 +21,7 @@ def pytest_addoption(parser):
         help="If provided, will not annihilate the data generated.")
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture()
 def workspace(request):
     # TODO: Use pytest.config.option('--workspace')
     test_workspace = tempfile.mkdtemp()
```

Comment:
What component actually does the deleting, then? That's not what my intuition about how this works is telling me.